docs/guides/integration-prebid-server-side.md

```diff
@@ -0,0 +1,415 @@
+
+For an example, see [Sample Token](#sample-token).
+
+### Refreshing a UID2 Token
+
+There are two ways to refresh a UID2 token, as shown in the following table.
+
+| Mode | Description | Link to Section | 
+| --- | --- | --- |
+| Client refresh mode | Prebid.js automatically refreshes the tokens internally.<br/>This is the simplest approach. | [Client Refresh Mode](#client-refresh-mode) |
+| Server-only mode | Prebid.js doesn't automatically refresh the token. It is up to the publisher to manage token refresh.<br/>Examples of why you might want to choose this option:<ul><li>If you want to use the [UID2 SDK for JavaScript](../sdks/client-side-identity.md) to refresh the token, and Prebid.js to send the token to the bid stream.</li><li>If you want to send the token to the bid stream via multiple avenues (such as Prebid.js and also Google).</li></ul> | [Server-Only Mode](#server-only-mode) |
+
+### Client Refresh Mode
+
+You must provide the Prebid module with the full JSON response body from the applicable endpoint:
+
+- [POST /token/generate](../endpoints/post-token-generate.md) for a new UID2 token.
+- [POST /token/refresh](../endpoints/post-token-refresh.md) for a refreshed UID2 token.
+
+For an example, see [Sample Token](#sample-token).
+
+As long as the refresh token remains valid, the UID2 Prebid module refreshes the UID2 token as needed.
+
+This section includes the following information:
+- [Client Refresh Mode Response Storage Options](#client-refresh-mode-response-storage-options)
+- [Client Refresh Mode Cookie Example](#client-refresh-mode-cookie-example)
+- [Client Refresh Mode uid2Token Example](#client-refresh-mode-uid2token-example)
+- [Passing a New Token: Client Refresh Mode](#passing-a-new-token-client-refresh-mode)
+
+#### Client Refresh Mode Response Storage Options
+
+When you configure the module to use Client Refresh mode, you must choose **one** of the following options for storing the API response information.
+
+| Option | Details | Use Case | 
+| --- | --- | --- |
+| Set `params.uid2Cookie` to the name of the cookie that contains the response body as a JSON string. | See [Client Refresh Cookie Example](#client-refresh-cookie-example). | Use this option only when there is enough space left in your cookie to store the response body. (**GWH_LP01 do we need to address how user will know if there is space in the cookie?**) (**also GWH_LP02 SW's question "is this optional even for when using local storage (i.e. not using cookie)?"**|
+| Set `params.uid2Token` to the response body as a JavaScript object. | See [Client Refresh uid2Token Example](#client-refresh-uid2token-example). | You might choose to provide the response body via `params.uid2Token` in either of these cases:<ul><li>If storing the response body on the cookie will exceed the cookie size limit.</li><li>If you prefer to have the flexibility to manage the storage of the response body yourself.</li></ul> |
```

Comment:
> If you prefer to have the flexibility to manage the storage of the response body yourself.

I'm not sure what this means. If anything, this option means they have less involvement in storing the response body. With the cookie option, they store the value. With this option, they don't store the value - the module does it for them.

> If storing the response body on the cookie will exceed the cookie size limit.

It's not really possible to know this for sure - cookie sizes change, and sites usually try to leave some "wiggle room". I would prefer something like:

> If you are already storing a lot of data in the cookie and adding the response body might exceed the cookie size limit.

Reply:
@lionell-pack-ttd thx. Updated to:
-     If you are already storing a lot of data in the cookie and adding the response body might exceed the cookie size limit.
-     If you prefer to have the the Prebid module store the token value for you.